release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1947,6 +1947,8 @@ This update fixes the problem by introducing client-go 1.22.
 
 * Previously, pod namespaces managed by CRI-O, for example network, IPC, and UTS, were not unmounted when the pod was removed. This resulted in leakage, driving the Open vSwitch CPU to 100%, which caused pod latency and other performance impacts. This has been resolved and pod namespaces are unmounted when removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2003193[*BZ#2003193*])
 
+* Currently, containers are started with non-empty inheritable Linux process capabilities. To work around this issue, modify the entry point of a container using a utility such as `capsh(1)` to drop inheritable capabilities before the primary process starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076265[*BZ#2076265*])
```

Comment:
```suggestion
* Currently, containers start with non-empty inheritable Linux process capabilities. To work around this issue, modify the entry point of a container using a utility such as `capsh(1)` to drop inheritable capabilities before the primary process starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076265[*BZ#2076265*])
```

Wording suggestion to avoid passive voice. WDYT?

Reply:
Good suggestion, Ben. I will implement and then ask for merge review if that is OK.